src/unity/toolkits/drawing_recognition/data_preparation.hpp

```diff
@@ -0,0 +1,43 @@
+/* Copyright © 2017 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+ */
+#ifndef TURI_DRAWING_DATA_PREPARATION_H_
+#define TURI_DRAWING_DATA_PREPARATION_H_
+
+#include <export.hpp>
+#include <unity/lib/gl_sframe.hpp>
+#include <unity/lib/variant.hpp>
+#include <unity/lib/image_util.hpp>
+#ifdef __APPLE__
+#include <CoreGraphics/CoreGraphics.h>
```

Comment:
Nothing in this header uses CoreGraphics; please remove the include.

Reply:
Moved it to data_preparation.cpp